### ISSUE TITLE
Keyboard focus is not properly visible for the drop down control

### ISSUE THREAD

**Issue by Ram-G** [Azure/LogicAppsUX#3550]
### Describe the Bug with repro steps

1. Create a new flow with designer standalone (nx serve).
2. Add an "Initialize variable" action.
3. Click on dropdown for "Type" field
4. Observe that when a menu item is highlighted in the drop down, it has insufficient contrast with background.

### What type of Logic App Is this happening in?

Consumption (Portal)

### Are you using new designer or old designer

New Designer

### Did you refer to the TSG before filing this issue? https://aka.ms/lauxtsg

Yes

### Screenshots or Videos

![image](https://github.com/Azure/LogicAppsUX/assets/10837129/7ef1aa8b-b67c-463a-b30b-cfb9a44f14ea)

### Browser

Edge

### Additional context

Related internal bug: https://msazure.visualstudio.com/One/_workitems/edit/13413625

AB#25622164

**Comment by hartra344** [comment]
This is the color choice by FluentUI by default, this was brought up on their Github and the resolution they gave was that the hover background did not need to conform to the 3:1 rule.

[10464](https://github.com/microsoft/fluentui/issues/10464)

**Comment by Ram-G** [comment]
This bug appears to be not repro as a black bounding box appears around the currently selected dropdown control (checked in nx serve). Closing this issue.

![image](https://github.com/Azure/LogicAppsUX/assets/10837129/be486555-e42a-40a1-b074-0cf9d6831ba9)
